Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -158,12 +158,9 @@ private void HandlePausePressed()
             switch (_stateManager.GetState())
             {
                 case State.Play:
-                {
                     _stateManager.SetState(State.Pause);
                     break;
-                }
                 case State.Pause:
-                case State.Tutorial:
```

Comment:
There was a bug where you can press escape, and it'll also show the pause screen. 

This disables pressing escape key